README.tpl.md

```diff
@@ -77,7 +77,7 @@ For an out-of-the-box setup, you can use my docker-compose recipe or a docker im
 
 For automating the analysis of your Scala projects, check out my sbt plugin [mwz/sbt-sonar](https://github.com/mwz/sbt-sonar).
 
-Also, see the [examples](https://github.com/mwz/sonar-scala/tree/master/examples) directory, which includes sample projects for sbt, Gradle and Maven along with basic instructions on how to execute SonarQube analysis for each of those projects.
+Also, see the [examples](https://github.com/mwz/sonar-scala/tree/master/examples) directory, which includes sample projects for SBT _(`1.x`)_, Gradle _(`5.x`)_ and Maven _(`3.x`)_ along with basic instructions on how to execute SonarQube analysis for each of those projects.
```

Comment:
I am aware this is probably not the best way to show with which versions we run the examples...
But right now I can barely stay awake 😅 

If you have better alternatives I would be happy to read your suggestions.

Reply:
I think that this is ok.

Reply:
Ok :+1:

Update, I realize I used an older version of **Scapegoat** _(`1.3.3`)_ instead of latest _(`1.3.8`)_.

So, I am updating it, but the update produces an error, so please do not merge this until the next commit 😉

Reply:
The error is from **Scapegoat** _(sksamuel/scapegoat#226)_.

Reply:
Ok so, I just decided to leave it in `1.3.8` even if they fail now, and as soon as scapegoat releases a new `1.3.9` version open a PR upgrading it on all places here: Main project, meta-project and sbt, gradle & mvn examples _(mvn PR coming soon 😉)_.

What do you think?
The other option would be to downgrade to `1.3.6` which was the last one that worked.

Reply:
> Ok so, I just decided to leave it in `1.3.8` even if they fail now, and as soon as scapegoat releases a new `1.3.9` version open a PR upgrading it on all places here

I'm happy with either, whatever is more convenient for you, however, downgrading would be a safer option as we could merge those PRs without having to wait for a new release - otherwise, we would end up with a broken pipeline.

Reply:
Yeah, I was thinking about that too.
So, I will downgrade it to do not break the build.